examples/nodejs-local/js/webpack.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (config) => {
+    // turn on __dirname replacement so that the WASM file can be referenced by the assemblyscript lib
+    config.node = {
+        __dirname: true,
+    }
+    // set the "context" for where __dirname should be relative to, to the root of the repo.
+    config.context = config.context + '/../../..'
```

Comment:
This will never backfire for sure

Reply:
definitely.

Reply:
i figure it's lower risk because its the example app webpack and not the actual SDK bundle